MAINTAINERS.md

```
@@ -0,0 +1,23 @@
+
+## Mentors
+
+| Name | GitHub Username | Discord Username | Area / Focus                        |
+| ---- | --------------- | ---------------- | ----------------------------------- |
+| TODO | @username       | @discord_user    | whole Project Guidance & PR Reviews |
+
+## Maintainers
+
+| Name | GitHub Username | Discord Username | Area / Focus                     |
+| ---- | --------------- | ---------------- | -------------------------------- |
+| TODO | @username       | @discord_user    | Repository Maintenance & Merging |
```

markdownlint-cli2 (0.23.2)

[warning] Files should end with a single newline character

(MD047, single-trailing-newline)